lib/media/segment_reference.js

```diff
@@ -160,8 +160,8 @@ shaka.media.SegmentReference = class {
    *  If not provided, the duration should be automatically calculated based on
    *  the duration of the reference.
    * @param {?number=} syncTime
-   *  A time value, expressed in the same scale as the start and end time, which
-   *  is used to synchronize between streams.
+   *  A time value, expressed in seconds since 1970, which is used to
```

Comment:
Probably worth mentioning that is is produced AND consumed inside the manifest parser, to make it clearer that this is something that doesn't really need to be worried about by other components.

Reply:
Done!